dev-docs/publisher-api-reference.md

```diff
@@ -715,7 +719,7 @@ pbjs.addAdUnits({
             sizes: [[300, 250], [300, 600]]
```

Comment:
looks like this adUnit has both adUnit.sizes as well as adUnit.mediaTypes.banner.sizes.

Reply:
Thanks, addressed by cefb91e